spec/index.bs

```diff
@@ -1048,9 +1048,14 @@ on all implementations, typically due to hardware or system software constraints
 Each {{GPUAdapter}} exposes a set of available features.
 Only those features may be requested in {{GPUAdapter/requestDevice()}}.
 
-Functionality that is part of an feature may only be used if the feature
-was requested at device creation. See the [[#feature-index|Feature Index]]
-for a description of the functionality each feature enables.
+Functionality that is part of an feature may only be used if the feature was requested at device
+creation. Dictionary members added to existing dictionaries by optional features must have their
+default values (which occurs implicitly) if the feature is not enabled.
+
+Enabling a feature on a device will not break code that works on a device without that feature.
```

Comment:
This one is less obvious than it seems at first. In Vulkan land, enabling an extension *can* make your program invalid. So while the wording is obvious, it doesn't match the existing solutions or practice.

Reply:
Is this something we would run into? I can imagine it making sense with, say, vulkan portability, but personally I would put something like that (webgpu compat) somewhere else in the device descriptor rather than calling it a "feature".

Reply:
Possibly? I could think of cases where we'd need that, like a feature that brings a new usage flag. We'd suddenly start validating against the new flag, and old applications may break just because they enable the feature.

Generally speaking, enabling features doesn't have to be backwards compatible because the act of enabling is an explicit opt-in, it's a consent to potential changes in the behavior.

That is to say, we can just promise to each other to not have such features that would break anything. That's a decent path forward, but it should be a conscious choice by the group, not merely a part of a small fix for error conventions.

Reply:
Makes sense. I'm thinking of the case where you have libraries and middleware and you don't want to break them by enabling a feature. I guess we should consider this on a case by case basis.

Reply:
Fixed